Formula/libimobiledevice.rb

```diff
@@ -13,27 +13,73 @@ class Libimobiledevice < Formula
   end
 
   head do
-    url "http://git.sukimashita.com/libimobiledevice.git"
+    url "https://git.libimobiledevice.org/libimobiledevice.git"
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
     depends_on "libxml2"
   end
 
+  option "with-python", "Enable Cython Python bindings"
+  option "with-python3", "Enable Cython Python3 bindings"
+
+  libplist_depends = []
+  libplist_depends << "with-python" if build.with? "python"
+  libplist_depends << "with-python3" if build.with? "python3"
+
   depends_on "pkg-config" => :build
   depends_on "libtasn1"
-  depends_on "libplist"
   depends_on "usbmuxd"
   depends_on "openssl"
+  depends_on "libplist" => libplist_depends
+  depends_on :python => :optional
+  depends_on :python3 => :optional
+
+  resource "cython" do
+    url "https://github.com/cython/cython/archive/0.24.tar.gz"
+    sha256 "b60b91f1ec88921a423d5f0a5e2a7c232cdff12d9130088014bf89d542ce137b"
+  end
+
+  patch do
+    # ensure python3 compatibility
+    url "https://github.com/libimobiledevice/libimobiledevice/pull/335.patch"
```

Comment:
I do not expect anything happening there, the maintainers are not very active on github. Should we close this PR here because of this?